### PR TITLE
Insertdeparser

### PR DESCRIPTION
**Pull request by binkertpat**

### CODE REVIEW

File: src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java [hunk 1]
```diff
@@ -440,16 +441,18 @@ public void visit(Column tableColumn) {
...
+                tableName += table.getAlias().toString();
             }
-        }
-
-        if (!tableName.isEmpty()) {
-            buffer.append(RegExGenerator.useSpellingMistake(this.tableNameSpellingMistake, tableName));
-            buffer.append('.');
+            tableName =
+                    "(?:"
+                    + RegExGenerator.useSpellingMistake(this.columnNameSpellingMistake, tableName)
```
Comment (Alletkla) [inline src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java:451]: Arent there now to 2 settings mixed up? Column and TableNameSpelling?

File: src/test/java/sqltoregex/deparser/TestUtils.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Component
+public class TestUtils extends UserSettingsPreparer{
+    StatementDeParserForRegEx statementDeParserForRegEx;
+
+    @Autowired
+    public TestUtils() throws XPathExpressionException, ParserConfigurationException, IOException, URISyntaxException, SAXException {
```
Comment (Alletkla) [inline src/test/java/sqltoregex/deparser/TestUtils.java:26]: Why this needs Autowiring & Component Annotation?

File: src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java (hunk 1 above)
Comment (binkertpat) [reply src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java:451]: oh my fault

File: src/main/java/sqltoregex/deparser/OrderByDeParserForRegEx.java [hunk 3]
```diff
@@ -47,79 +42,99 @@ public OrderByDeParserForRegEx(ExpressionVisitor expressionVisitor, StringBuilde
...
+                temp.append(RegExGenerator.useSpellingMistake(this.keywordSpellingMistake, "NULLS LAST"));
+            }
+        }
+        return temp.toString();
+    }
+
+    public String deParseElementForOrderRotation(OrderByElement orderByElement, FromItem fromItem){
+        StringBuilder temp = new StringBuilder();
+        if(orderByElement.getExpression().toString().contains("(") && orderByElement.getExpression().toString().contains(")")){
+            temp.append(RegExGenerator.useStringSynonymGenerator(this.aggregateFunctionLang, orderByElement.getExpression().toString().replaceAll("\\(.*", "")));
```
Comment (Alletkla) [inline src/main/java/sqltoregex/deparser/OrderByDeParserForRegEx.java:98]: Why an order by element toString results in Regex? \\(.*

File: src/test/java/sqltoregex/deparser/TestUtils.java (hunk 2 above)
Comment (binkertpat) [reply src/test/java/sqltoregex/deparser/TestUtils.java:26]: not needed anymore

**Review by Alletkla — APPROVED** [review]

**Pull request merged**